digits/model/images/classification/views.py

```diff
@@ -267,6 +267,7 @@ def create():
                         pretrained_model = pretrained_model,
                         crop_size = form.crop_size.data,
                         use_mean = form.use_mean.data,
+                        mean_path = form.mean_path.data,
```

Comment:
I'd rename this to e.g. `custom_mean_path` since it could be confusing to have a `mean_path` field set to `None` when we are using the dataset mean.